test/distributions/block/arrayAssignment.good

```diff
@@ -0,0 +1,3 @@
+newSharedRegion 35
+newSharedRegion 37
+newSharedRegion 43
```

Comment:
These line numbers look off to me.  Shouldn't it be `31 33 35`?

Reply:
It seems we had a race to the same conclusion :). This is already fixed

Reply:
Can I add "almost as good at detecting errors as running the test itself" as one of my epithets?  XD